### PR TITLE
Made edits to the zipcode regex

### PR DESCRIPTION
**Pull request by AkeilaHarris**

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
I'm approving this, however, I would have left your original test cases the way they were - they were definitely valid - The root problem here is that you didn't have any test cases which looked for the zip code without any other text.  I would have added them as additional tests and not removed your original ones.

**Comment by claesmk** [comment]
@AkeilaHarris this is waiting on a review from someone else on your team.  After that it should merge OK

**Comment by claesmk** [comment]
@AkeilaHarris reminder that you still need someone from your team to approve your PR so you can merge it

**Review by ashadixon — APPROVED** [review]

**Pull request merged**
